### PR TITLE
[bug 1248040] Update GTM data attributes on homepage navigation and F…

### PR DESCRIPTION
**Pull request by JamesDLorence**
[bug 1248040] - https://bugzilla.mozilla.org/show_bug.cgi?id=1248040

Added data attributes to the homepage navigation, the Get Involved with Mozilla today button and the See all events button on the homepage, and to the Firefox download buttons.

Deleted media/js/mozorg/home/ga-tracking.js as we no longer need those data attributes added to the download buttons.

### CODE REVIEW

File: bedrock/firefox/templates/firefox/includes/download-button.html [hunk 1]
```diff
@@ -43,7 +43,9 @@
       <li class="os_{{ plat.os }}{% if plat.arch %} {{ plat.arch }}{% endif %}">
         <a class="download-link"
            href="{{ plat.download_link }}"{% if plat.download_link_direct %}
-           data-direct-link="{{ plat.download_link_direct }}"{% endif %}>
+           data-direct-link="{{ plat.download_link_direct }}"{% endif %}
+           {% if plat.os == 'android' %}data-link-type="download" data-download-os="Android"
+           {% else %}data-link-type="download" data-download-os="Desktop"{% endif %}>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/includes/download-button.html]: This is incorrectly adding `data-download-os="Desktop"` to iOS download links as well as desktop.

File: bedrock/mozorg/templates/mozorg/home/home.html [hunk 2]
```diff
@@ -88,16 +88,16 @@
         <span class="toggle" role="button" aria-controls="nav-main-menu" aria-expanded="false" tabindex="0">{{_('Menu')}}</span>
         <ul id="nav-main-menu">
           <li class="first about-item">
-            <a data-link-type="about" data-element-location="nav click" href="{{ url('mozorg.about') }}">{{_('About')}}</a>
+            <a href="{{ url('mozorg.about') }}" data-link-type="nav" data-link-name="About">{{_('About')}}</a>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/home/home.html]: Changing these data attributes breaks one of our functional tests. Can you please update the selectors here to match these changes? https://github.com/mozilla/bedrock/blob/master/tests/pages/base.py#L42-L44

**Comment by alexgibson** [comment]
When deleting a JS file, please also make sure to remove it from the relevant page bundle here: https://github.com/mozilla/bedrock/blob/master/bedrock/settings/static_media.py#L1417

File: bedrock/mozorg/templates/mozorg/home/home.html (hunk 2 above)
Comment (JamesDLorence) [inline bedrock/mozorg/templates/mozorg/home/home.html]: I'll make the updates. I didn't see any failing tests when I ran `./manage.py test` in the terminal. Is that a concern?

Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/home/home.html]: `.manage.py test` just runs the python unit tests. See here for how to run the functional tests if you need to: http://bedrock.readthedocs.org/en/latest/testing.html

**Comment by alexgibson** [comment]
Code changes look good, r+

Because this changes locators that are used in our functional tests, we'll need to time merging this with a production deployment. Will try and merge/deploy this later today. Thanks @JamesDLorence!

**Pull request merged**